paper-server/src/main/java/org/bukkit/craftbukkit/inventory/view/builder/CraftBlockEntityInventoryViewBuilder.java

```diff
@@ -44,23 +51,36 @@ protected AbstractContainerMenu buildContainer(final ServerPlayer player) {
             return buildFakeTile(player);
         }
 
+        trySetDefaultTitle(entity);
```

Comment:
Less of a `try`, this will error if it doesn't succeed and that error isn't an expected state unless we fucked up.
So just, `setDefaultTitle`, or honestly `super.defaultTitle = extractDefaultTitle(entity)` is better here.